api/src/convert/mod.rs

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2018-2020 MobileCoin Inc.
+
+//! Conversions between "API types" and "domain/persistence types".
+//!
+//! gRPC and Protobuf provide a reduced selection of types, and so there are some differences between
+//! values stored in the ledger and values transmitted over the API. This module provides conversions
+//! between "equivalent" types, such as `mc_api::blockchain::Block` and `mc_transaction_core::Block`.
+
+mod error;
+
+// blockchain
```

Comment:
Splitting things out as "blockchain" conversions vs. "external" conversions seems to mirror the gRPC definitions.